### DISCUSSION TITLE
How to use Aws Api Gateway Websocket

### DISCUSSION THREAD

**Discussion by msimonc** [socketio/socket.io#3790] [Q&A]
wscat -c wss... connects w/o extra security configs,

but ReactJS client "blocked by CORS policy" on io()

I'm hoping for wss 

the error also is not thrown by the method so I must set a timeout then disconnect()....

**Comment by darrachequesne** [comment]
If I understand correctly, you need to only enable the WebSocket transport:

```js
const socket = io("https://example.com", { transports: ["websocket"] });
```

Reference: https://socket.io/docs/v3/client-initialization/#transports

You could also configure CORS on the server: https://socket.io/docs/v3/handling-cors/
